engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -215,8 +220,11 @@ class WorkflowManagerActor(config: Config,
     // Uninteresting transition and current state notifications.
     case Event((Transition(_, _, _) | CurrentState(_, _)), _) => stay()
     case Event(JobStoreWriteSuccess(_), _) => stay() // Snoozefest
-    case Event(JobStoreWriteFailure(_, t), _) =>
-      log.error("Error writing to JobStore from WorkflowManagerActor: {}", t)
+    case Event(JobStoreWriteFailure(failedOperation @ RegisterWorkflowCompleted(wfId), t), _) =>
+      log.error("Error registering JobStore completion for {}: {}", arg1 = wfId, arg2 = t)
+      // This is minorly bad. The JobStore would never rid itself of this workflow's entries. Unlikely to be a big deal
+      // but might as well try again in a few minutes...
+      context.system.scheduler.scheduleOnce(10 minutes, sender, failedOperation)(context.dispatcher)
```

Comment:
I'm a little concerned about cruft building up for reasons which would actually alarm us but we never notice. OTOH firing a log message is probably not enough for us to notice it is happening either so YMMV